src/main/java/seedu/address/model/animal/Animal.java

```diff
@@ -108,4 +114,7 @@ public String toString() {
         return builder.toString();
     }
 
+    public Set<FeedTime> getFeedTimes() {
+        return Collections.unmodifiableSet(feedTimes);
+    }
```

Comment:
I suggest you can add a similar javadoc comment for this method by following the getMedicalConditions() method above (since it is provided already).

Reply:
Javadoc added